CMakeLists.txt

```diff
@@ -190,6 +190,10 @@ if(BUILD_SHARED_LIBS)
     set_target_properties(h3 PROPERTIES SOVERSION ${H3_SOVERSION})
 endif()
 
+if(MSVC)
```

Comment:
I'd prefer not adding this because it introduces some marginal build complexity, but the tests are not a performance critical path.